### PR TITLE
Update pull-request-guidelines.md

### PR DESCRIPTION
**Pull request by chawes13**

### CODE REVIEW

File: gists/pull-request-guidelines.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Guidelines for creating pull requests at LaunchPad Lab for three types of users:
 - Provide screenshots and instructions to reproduce / find style issues if there are outstanding items that require design assistance
 - Add guidance about what the reviewer(s) should focus on
 - Document what the QA tester should be testing
+- Review the diff and ensure that all changes are relevant and intentional
```
Comment (inveterateliterate) [inline gists/pull-request-guidelines.md:19]: one other thing that comes to mind, "relevant and intentional, and address requirements of listed issue or ticket" or something along those lines

Comment (inveterateliterate) [reply gists/pull-request-guidelines.md:19]: this is getting at both - did you cover off on the requirements of the ticket? (other than what's out of scope) and / or are there items in this diff that don't pertain to the requirements the PR or branch was meant to address

**Review by inveterateliterate — APPROVED** [review]
love it!

**Comment by bhennes2** [comment]
🎉

**Pull request merged**